ecc/bls12381/g2.go

```diff
@@ -187,10 +187,84 @@ func (g *G2) clearCofactor() {
 }
 
 // Double updates g = 2g.
-func (g *G2) Double() { doubleAndLine(g, nil) }
```

Comment:
Why is this changed?

Reply:
this is a stale commit, not needed. Fixed.